test/makeCompleteAction.test.js

```diff
@@ -161,4 +161,84 @@ describe("makeCompleteAction", () => {
       });
     });
   });
+
+  describe("with customSuccessPayload option", () => {
+    const customSuccessPayload = payload => payload;
```

Comment:
Since you're using this same function in the `customSuccessPayload`, `customFailurePayload` (and in the `customRequestPayload` in the future), you could rename it to a more general name like `customPayload`, create a `describe` function that wraps these three test cases, put this function inside this `describe` function and use it in the three custom payloads `describe`.

You can use it like this, for example: `customSuccessPayload: customPayload`.

I'm not sure if I explained myself well, let me know!

Reply:
made perfect sense, I've made the change